docs/source/main_classes/deepspeed.rst

```diff
@@ -1555,9 +1574,60 @@ stress on ``tensor([1.])``, or if you get an error where it says the parameter i
 larger multi-dimensional shape, this means that the parameter is partitioned and what you see is a ZeRO-3 placeholder.
 
 
+
+
+Filing Issues
+=======================================================================================================================
+
+Here is how to file an Issue so that we could quickly get to the bottom of the Issue and help you to unblock your work.
```

Comment:
```suggestion
Here is how to file an issue so that we could quickly get to the bottom of the issue and help you to unblock your work.
```
No need for capitals IMO.